cmd/nirvana/api/api.go

```diff
@@ -77,30 +77,27 @@ func (o *apiOptions) Install(flags *pflag.FlagSet) {
 }
 
 func (o *apiOptions) Validate(cmd *cobra.Command, args []string) error {
+	if len(args) < 1 {
+		return fmt.Errorf("must specify a project path")
+	}
 	return nil
 }
 
 func (o *apiOptions) Run(cmd *cobra.Command, args []string) error {
-	builder := utils.NewAPIBuilder(o.findAllChildernPaths(args...)...)
+	builder := builder.NewAPIBuilder(project.Subdirectories(false, args...)...)
 	definitions, err := builder.Build()
 	if err != nil {
 		return err
 	}
-	file := ""
+	config := &project.Config{}
 	for _, path := range args {
-		file, err = o.findProjectFile(path)
+		config, err = project.LoadDefaultProjectFile(path)
```

Comment:
Since it will use the first configuration that can be read, I think it is possible to limit the length of args to not greater than 1